### DISCUSSION TITLE
Mysql Version

### DISCUSSION THREAD

**Discussion by PraveenKumarAdabala** [LiveHelperChat/livehelperchat#1810] [Q&A]
Hi Team,

We are upgrading lhc 2.91v  to latest in our application, we would like to know which version of mysql we can use with the latest lhc.
You have already specified to use version > 5.7. But we would like to know which specific version you have used to test the lhc application so that we will use the same version.

Thanks,
Praveen

**Comment by remdex** [comment]
I'm using 5.5.5-10.3.35-MariaDB. DB has to support json_extract syntax. That's the only requirement.
